redwood-layout-api-density/src/androidMain/kotlin/app/cash/redwood/layout/api/density.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.layout.api
+
+import android.content.res.Resources
+
+// Android uses 2.75 as a density scale for most recent Pixel devices and iOS
+// uses 3. This aligns the two so the generic values used by Redwood layout are
+// visually similar on both platforms.
+internal actual val DensityMultiplier = 1.1
```

Comment:
There's probably a better way to roughly normalize iOS and Android densities, but this is what we've been doing so far.

Reply:
Hmmm, is there a reason why we'd want this? My gut feeling is that we should just abide from the platform choice.

Reply:
@chrisbanes It's to normalize the density values between iOS and Android (and other future platforms). Without a multiple, `4.dp` looked much smaller on Android than iOS.

Reply:
This is tricky because that's mostly an intentional choice for the Android OS. I'm not sure we should be getting into the game of creating 1:1(ish) pixel consistency.

WDYT?

Reply:
@chrisbanes That's a good point and we probably need to chat with product + design about it, though my hunch is that they'll want similar sizes on both platforms.

I vote we keep `DensityMultiplier` as an implementation detail that we can change later if necessary, but keep it as is for the moment since we have existing screens built with this assumption.

Reply:
Sounds good to me!